02-express-tutorial/03-express-basics.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express')
+const app = express()
+
+app.get('/', (req, res) => {
+  console.log('user hit the resource')
```

Comment:
This looks like a lingering debugging message; make sure to avoid leaving those behind, or use a debugger to avoid adding lots of `console.log`s entirely!